packages/jest-circus/src/eventHandler.ts

```diff
@@ -21,7 +21,10 @@ import {
   restoreGlobalErrorHandlers,
 } from './globalErrorHandlers';
 
-const eventHandler: Circus.EventHandler = (event, state): void => {
+const eventHandler: Circus.EventHandler = (
+  event: Circus.Event,
```

Comment:
same as above

Reply:
@SimenB, here I cannot apply your suggestion due to the error from TypeScript compiler:

```
$ yarn build:ts
$ node ./scripts/buildTs.js
 Building TypeScript definition files 
packages/jest-circus/src/eventHandler.ts:24:44 - error TS7006: Parameter 'event' implicitly has an 'any' type.

24 const eventHandler: Circus.EventHandler = (event, state): void => {
                                              ~~~~~

packages/jest-circus/src/eventHandler.ts:24:51 - error TS7006: Parameter 'state' implicitly has an 'any' type.

24 const eventHandler: Circus.EventHandler = (event, state): void => {
                                                     ~~~~~
```

Can we leave the code here as-is? Or you strongly wish to find some other solution? I am just afraid I cannot suggest anything else except to start debugging the TypeScript compiler, which looks very much out of the scope. :confused:

Reply:
very strange! feel free to leave, but maybe include a todo comment to investigate at some point?

Reply:
Okay, sure! I have added this TODO comment. Please tell me if it is good enough:

```diff
+// TODO: investigate why a shorter (event, state) signature results into TS7006 compiler error
 const eventHandler: Circus.EventHandler = (
   event: Circus.Event,
   state: Circus.State,
```

Reply:
yup, that looks good 👍